src/core/database/config/config.js

```diff
@@ -0,0 +1,40 @@
+const fs = require('fs');
+
+module.exports = {
+  development: {
+    username: process.env.MY_SQL_USER || 'root',
```

Comment:
MYSQL is a single word. Let's also update the name in the other places
```suggestion
    username: process.env.MYSQL_USER || 'root',
```